lib/pubspec_parser.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:io';
+
+import 'package:yaml/yaml.dart';
+
+/// helper class for parsing the contents of pubspec file
+class PubspecParser {
+
+  /// ensures unnamed constructor cannot be used as this class should only have
+  /// static methods
+  PubspecParser._();
+
+  /// parses the pubspec located at [path] to map
+  static Map toMap(String path) {
```

Comment:
I suggest to rename `toMap()` function to `fromFSPath` or `fromPath` or `loadFromPath`

Reply:
What about `fromPathToMap(...)`?

Reply:
Even better

Reply:
Done!